extensions/android/package.json

```diff
@@ -0,0 +1,67 @@
+{
+  "$schema": "https://www.raycast.com/schemas/extension.json",
+  "name": "android",
+  "title": "Android",
+  "description": "Useful Android commands",
+  "icon": "command-icon.png",
+  "author": "mibrahimdev",
+  "categories": [
+    "Developer Tools"
+  ],
+  "license": "MIT",
+  "commands": [
+    {
+      "name": "list-android",
+      "title": "List Projects",
+      "subtitle": "Android",
+      "description": "show android projects in directory",
+      "icon": "list-android.png",
+      "mode": "view"
+    },
+    {
+      "name": "list-emulators",
+      "title": "List Emulators",
+      "subtitle": "Android",
+      "description": "show android emulators",
+      "icon": "list-android.png",
+      "mode": "view"
+    }
+  ],
+  "dependencies": {
+    "@raycast/api": "^1.39.0"
+  },
+  "devDependencies": {
+    "@typescript-eslint/eslint-plugin": "^5.33.1",
+    "@typescript-eslint/parser": "^5.33.1",
+    "eslint": "^7.32.0",
+    "eslint-config-prettier": "^8.3.0",
+    "eslint-plugin-react": "^7.30.1",
+    "prettier": "^2.5.1",
+    "typescript": "^4.4.3"
+  },
+  "preferences": [
+    {
+      "title": "Android Projects Directory",
+      "name": "androidProjectsDirectory",
+      "description": "Locate your Android Projects, without '/'",
```

Comment:
Without what? Maybe also add a placeholder as an example.

Maybe you should also use a library like `expand-tilde`